src/Compiler/CompilerOptions.cpp

```diff
@@ -0,0 +1,257 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===------------------------ CompilerOptions.cpp -------------------------===//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// Functions for adding options.
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/Support/Debug.h"
+
+#include "ExternalUtil.hpp"
+#include "onnx-mlir/Compiler/OMCompilerTypes.h"
+#include "src/Compiler/CompilerOptions.hpp"
+
+#define DEBUG_TYPE "compiler_options"
+
+using namespace std;
+using namespace onnx_mlir;
+
+llvm::cl::OptionCategory OnnxMlirOptions(
+    "ONNX-MLIR Options", "These are frontend options.");
+
+namespace onnx_mlir {
+
+// This definition is here rather than in main.cpp because otherwise it's not
+// found probably should be pulled out to a more common location
+// TODO: Find a respectable home for the wain
+
+// the option is used in this file, so defined here
```

Comment:
Not sure what this comment mean, remove ?

Reply:
Removed. This is is the new home.